### ISSUE TITLE
Question on time zone of dates/times

### ISSUE THREAD

**Issue by iita-atii** [LibraryOfCongress/api.congress.gov#296]
Hey, are dates excluding a timezone mark assumed to be eastern time? It seems most dates don't have a time zone except (from what i've seen) for all `updateDate` and amendment's `submittedDate`, `proposedDate`. 

Just want to make sure dates with `Z` are UTC time and those excluding timezone markings are assumed to be eastern time.

**Comment by apreiter18** [comment]
The times you reference are in UTC. For example:

On Congress.gov: https://www.congress.gov/event/118th-congress/house-event/117742?s=2&r=3
Congress.gov API: https://api.congress.gov/v3/committee-meeting/118/house/117742?api_key=INSERT_KEY

The time of the committee meeting is 10AM, which would be reflected as 3pm in the API since we are using UTC. This is applicable across collections.

**Comment by iita-atii** [comment]
Thanks for your reply, so just to be clear times served by the API are correctly time zone marked (through either `z` or `+`) and if a time zone mark is excluded they are assumed to be eastern time?

I make the eastern time assumption because Actions date/time seem to be in eastern time when comparing against the roll calls they mention. Are there any you know of that wouldn't be marked with a time zone that should be interpreted as UTC?
```
        {
            "actionCode": "H37100",
            "actionDate": "2016-09-13",
            "actionTime": "16:48:56",
            "recordedVotes": [
                {
                    "chamber": "House",
                    "congress": 114,
                    "date": "2016-09-13T20:48:56Z",
                    "rollNumber": 502,
                    "sessionNumber": 2,
                    "url": "https://clerk.house.gov/evs/2016/roll502.xml"
                }
            ],
            "sourceSystem": {
                "code": 2,
                "name": "House floor actions"
            },
            "text": "On passage Passed by the Yeas and Nays: 261 - 147 (Roll no. 502). (text: CR H5380)",
            "type": "Floor"
        },
```

**Comment by apreiter18** [comment]
Yes, you are correct. Here is an example of actions with EST timestamps: 

https://api.congress.gov/v3/bill/118/s/4477/actions?api_key=INSERT_KEY

and then on Congress.gov:
https://www.congress.gov/bill/118th-congress/senate-bill/4477/all-actions

The 10.41am and the 10.35am times are in EST and correspond to when the House of Representatives was in session.

**Comment by iita-atii** [comment]
Great, thanks for the clarity!
